conf.yaml

```
@@ -33,6 +33,9 @@ repos:
     go-elasticsearch:     https://github.com/elastic/go-elasticsearch.git
     elasticsearch-rs:     https://github.com/elastic/elasticsearch-rs.git
     elasticsearch:        https://github.com/elastic/elasticsearch.git
+    enterprise-search-pubs: https://github.com/elastic/enterprise-search-pubs.git
+    enterprise-search-python: https://github.com/elastic/enterprise-search-python
```

Even though this (apparently) works either way, for consistency I'd add .git extensions
```suggestion
    enterprise-search-python: https://github.com/elastic/enterprise-search-python.git
```